docs/onboarding.md

```diff
@@ -0,0 +1,116 @@
+# Service Team Onboarding
+
+This document outlines the steps required to onboard a new service and service
+team into the testing infrastructure. 
+
+## Prerequisites
+
+Before starting the onboarding process, the service team should have already
+created their service controller repository and have created a **_new_** AWS
+account solely for the purposes of hosting test resources.
+
+## 1. Configure [`ack-bot`][ack-bot] Access
+
+The [`ack-bot`][ack-bot] account exists for the Prow cluster to read and write 
+PR comments and to merge code into the `main` branch. The bot needs to be added
+individually to each Github repository, as it is not a member of the 
+`aws-controllers-k8s` organisation. 
+
+Contact the Amazon Open Source office to request the bot user be added to the 
+Github repository.
+
+[ack-bot]: https://github.com/ack-bot
+
+> **Note for Core Contributors:** Accept the invitation on behalf of the bot 
+account and change the bot to `Maintainer` role.
+
+## 2. Create an ACK test role
+
+When running e2e or soak tests, the test infrastructure will assume a role and
+create AWS resources in your account. In this section we will create that role 
+and provide access for the ACK infrastructure system. See the
+[IAM doc][iam-doc] for more details on how the system uses credentials.
+
+[iam-doc]: iam-structure.md
+
+### a. Create the base role
+
+In the AWS console, navigate to IAM and create a new Role. For now, choose
+`EC2` as the trusted entity. Initially select the `AdministratorAccess` 
+policy, although your team can adjust policies for stricter requirements in
+the future. Set the role name to `<service>-ack-test-role-DO-NOT-DELETE`.
```

Comment:
Any chance we can give the team an `aws` CLI command to execute here instead of describing the AWS console steps? Or is this going to be an Isengard-specific console thing?

Reply:
Yeah I only hesitated because I have to provide a trust policy JSON document alongside this documentation, which is a bit of a PITA. But will do it anyway